CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased] - ReleaseDate
 
+- Update dependencies
+
```

Comment:
Please do not edit this changelog directly, instead see [.changes](https://github.com/cross-rs/cross/tree/main/.changes#readme)